core/tests/test_opengraph.py

```diff
@@ -0,0 +1,55 @@
+from django.test import TestCase
+from django.contrib.staticfiles.testing import StaticLiveServerTestCase
+
+
+class OpengraphTagsTestCase(TestCase):
+    def test_opengraph_tags(self):
+        expected_description = (
+            "NYC experimental live music calendar • noise, improv, jazz, new music,"
+            " avant-electronics, weirdos • performance against corporate interests!"
+        )
+        expected_title = "nyc noise"
+        expected_domain = "nyc-noise.com"
+        expected_url = f"https://{expected_domain}/"
+        expected_image = (
+            f"{expected_url}static/core/images/NYC-Noise-facebook-preview-TOO-BIG.png"
```

Comment:
This is different in production, where whitenoise serves compressed file, couldn't make it happen in dev/test